java/spark-2.0.0/utils200.scala

```diff
@@ -0,0 +1,19 @@
+package sparklyr
```

Comment:
Minor, but can we just call this file `utils.scala`? (It's already implicitly versioned as it's part of the 2.0.0 folder). Or does the file name need to be the same as the object name for Scala?